content/examples/getting-started.md

```diff
@@ -1,144 +1,242 @@
 +++
 date        = "2014-05-07T11:27:27-04:00"
 title       = "Getting started"
-categories  = [ "examples" ]
 +++
 
-# Getting started
+# Quick Overview
 
-This document is a guide to introduce people to the `gb` project structure. A `gb` project is conceptually a `$GOPATH` per project, but saying that doesn't really help explain how to set up a new project; hence this document.
+This document is a guide to show you how to structure `gb` based Go projects and use `gb` to build those projects.
 
-# Creating an empty project
+A typical `gb` project may look like this on disk:
 
-A `gb` project is defined as any directory that has a `src/` subdirectory. The simplest possible `gb` project would be
+    /home/dfc/projects/engine/
+                        |——— src/
+                        |——— cmd/
+                              |——— engine/
+                                    |——— main.go
+                        |——— data/
+                        |——— services/
+                        |——— vendor/
+                              |——— src/
+                                    |——— github.com/codegangsta/cli/
+
+Since the `gb` tool can identify the workspace location of the project, imports can be referenced based on their location inside the `src/` and `vendor/src/` directories:
+
+    import (
+        "data"
```

Comment:
if these are packages, not folders at the root of the project, please fix the example output above